### PR TITLE
Updates to set bamboo environments for SBX and SIT

### PR DESCRIPTION
**Pull request by flamingbear**
Addresses [CUMULUS-bugfixes: Can't deploy to SIT](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2576)

# Updates to set bamboo environments for SBX and SIT

`set-bamboo-env-variables.sh` script updated to have all of the variables needed
to test and deploy the dashboard to SIT.

Bamboo modified to use script appropriately.

### Notes
Before, the bamboo tasks had explicit translations done in SIT e.g.:

export ESROOT=${bamboo.SIT_ESROOT}
export ES_USER=${bamboo.SECRET_SIT_ES_USER}
export ES_PASSWORD=${bamboo.SECRET_SIT_ES_PASSWORD}

Now running `. set-bamboo-env-variables.sh` in a task will parse those to the correct
values.

SIT_ESROOT =>  ESROOT
SECRET_SIT_ES_USER} => ES_USER
SECRET_SIT_ES_PASSWORD => ES_PASSWORD

## Changes

* Updates only the bamboo environment script
* Slips in an update to axios to fix a snyk error

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [x] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -79,6 +79,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - Fixes granule execute modal rerender issue when workflow options are not changed
 - **CUMULUS-2553**
   - Fixed DatePicker prop so the leading zeroes can be entered without having too many zeroes clouding the input.
+- **CUMULUS-NONE**
```
Comment (kkelly51) [inline CHANGELOG.md:82]: okay...

File: package.json [hunk 2]
```diff
@@ -149,7 +149,7 @@
     "@fortawesome/react-fontawesome": "^0.1.12",
     "@reduxjs/toolkit": "^1.5.0",
     "assert": "^1.4.1",
-    "axios": "^0.21.1",
+    "axios": "^0.21.4",
```
Comment (kkelly51) [inline package.json:152]: ahh I ran into this too. 
Wasn't there another one ? 
ansi-regex
Introduced through cypress@7.3.0,
Fixed in
ansi-regex@6.0.1, @5.0.1

**Review by kkelly51 — APPROVED** [review]
I'm not _that_ familiar with the bamboo script but it looks good to me

File: package.json (hunk 2 above)
Comment (flamingbear) [reply package.json:152]: Yes, but it didn't stop my PR so I didn't look into it.

**Pull request merged**